Gui/opensim/view/src/org/opensim/view/Installer.java

```diff
@@ -77,7 +80,21 @@ public boolean closing() {
 
     public void restored() {
         super.restored();
-        System.setProperty ("netbeans.buildnumber", "4.0.Beta"); // Should get that from JNI but sometimes doesn't work'
+        // The following block reads the file build_id.properties created during build/install
+        // and recovers netbeans.buildnumber to use it as a System property.
+        // Fomrat is YYYY-MM-DD timeInUTC abbreviated-commit-hash, will hsow on AboutJPanel 
```

Comment:
Is it true that this hash will show in the AboutJPanel? That would be great, because it would make it easier for testers/verifiers to list the "build ID" on GitHub. However, I don't see the code that actually causes the build ID to appear in the About panel.